examples/ContextDecorators/Program.cs

```diff
@@ -39,15 +42,19 @@ static void Main(string[] args)
         {
             var context = new LoggingRootDecorator(new RootContext());
             var props = Props.FromFunc(ctx =>
-            {
-                if (ctx.Message is string str)
                 {
-                    Console.WriteLine("Inside Actor: " + str);
-                    ctx.Respond("Yo!");
-                }
+                    if (ctx.Message is string str)
+                    {
+                        Console.WriteLine("Inside Actor: " + str);
+                        ctx.Respond("Yo!");
+                    }
+
+                    return Actor.Done;
+                })
+                .WithContextDecorator(c => new LoggingDecorator(c, "logger1"))
```

Comment:
Should we align this with WithReceiveMiddleware and make it variadic in terms of params?

Reply:
NVM, I see that it already is

Reply:
Well, I'm not sure it will do more.
By the way what is the goal of this code style (variadic funcs) in current middlewares ?